config/locales/errors/fr.yml

```diff
@@ -18,11 +18,25 @@ fr:
       token_missing: jeton manquant
     confirm_password_incorrect: Mot de passe incorrect.
     doc_auth:
+      acuant_network_error: Désolé, nous rencontrons des difficultés techniques. Veuillez
+        réessayer plus tard.
       acuant_throttle: To prevent abuse we limit the amount of times you can attempt
         to validate a document.  Please try again later.
       general_error: Votre identité émise par l'état n'a pas pu être vérifiée. S'il
         vous plaît essayez de télécharger des images différentes en commençant par
         le devant de votre ID émis par l'État.
+      general_info: |-
+        <div class="bold">Si vous utilisez une caméra, veuillez vous assurer que</div>
+        <ul class="mxn2 mt1 pl3 mb1 lg-ml2">
+        <li class="mxn3">Vous êtes dans une zone bien éclairée</li>
+        <li class="mxn3">Your ID does not have any damaged barcodes</li>
```

Comment:
If i recall correctly, we decided to let this go for now and fix it later?

Reply:
Hmm, did not notice this. Yeah, this won't be in front of any users, but let's open a bug for it.

Reply:
https://cm-jira.usa.gov/browse/LG-1269